cylc/uiserver/authorise.py

```diff
@@ -509,9 +510,12 @@ class AuthorizationMiddleware:
     def resolve(self, next_, root, info, **args):
         current_user = info.context["current_user"]
         # We won't be re-checking auth for return variables
-        if len(info.path) > 1:
+        if len(info.path.as_list()) > 1:
             return next_(root, info, **args)
```

Comment:
Might be worth adding a bit more detail to this comment, while we're here? Currently it is not very edifying to me

Reply:
Can you add a comment to explain what this line is achieving to ensure this context is remembered going forward.

Reply:
Ok, I've added a more detailed comment.